src/main/java/com/wcc/platform/domain/cms/attributes/CmsIcon.java

```diff
@@ -8,16 +8,18 @@
 @Getter
 @AllArgsConstructor
 public enum CmsIcon {
-  ICON_1("classIcon1"),
-  ICON_2("classIcon2"),
-  ICON_3("classIcon3"),
-  DIVERSITY_2("Diversity2");
+  BOOK("book_2"),
```

Comment:
This will be the google icons names, they will be used to decided which one it will be show on frontend. They are the available ones that frontend will use based on the figma template